### ISSUE TITLE
Feature request: highlight the cell with the best price in each row

### ISSUE THREAD

**Issue by zgerlock** [mikebryant/ac-nh-turnip-prices#124]
While it's not too hard to parse through them it would be nice if the peaks were highlighted since that's the main thing you care about. Would also serve to visually break up what is otherwise a huge grid of numbers. You could also highlight days of guaranteed profit one color and the peak another, that seems even better.

ps. thanks all for putting this together, it's dope. Love the nook phone styling

**Comment by mikebryant** [comment]
We now have some colours!

I think that satisfies this issue

And thanks for your comments. Glad you're liking the app :)
